src/cards/gist-card.js

```diff
@@ -77,8 +78,9 @@ const renderGistCard = (gistData, options = {}) => {
     .join("");
 
   const lineHeight = descriptionLines > 3 ? 12 : 10;
-  const height =
-    (descriptionLines > 1 ? 120 : 110) + descriptionLines * lineHeight;
+  const height = card_height
```

Comment:
Here, it would be nice to implement some logic to ensure that the card height is capped at the minimum height needed to display the contents. If I use `height=100` the card is messed up 🤔.

![image](https://github.com/anuraghazra/github-readme-stats/assets/17570430/511620f0-3c47-4474-b346-f43f06b70120)

Reply:
I'm not sure I push a new version do you want the calcul of the min height or you want a fix height :) ?

Reply:
@DanneelsSophie, I am afraid the minimum heights need to be calculated dynamically for all cards since the parameters will influence the amount of text on the card. I have not yet investigated the best way to do this.

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)
```

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra)](https://github.com/anuraghazra/github-readme-stats)

```md
[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show=reviews,discussions_started,discussion_answered,prs_merged,prs_merged_percentage)
```

[![Anurag's GitHub stats](https://github-readme-stats.vercel.app/api?username=anuraghazra&show=reviews,discussions_started,discussion_answered,prs_merged,prs_merged_percentage)

Reply:
Do you review the new version ? ( I must test this the calculated height it is min height). I would like this code structure is well for you !

Reply:
I will review a new version. However, please bear in mind that calculating the minimum required card might be a challenging task. Also, see @qwerty541's comment at https://github.com/anuraghazra/github-readme-stats/issues/3159#issuecomment-1780598362.

Reply:
I've kept the calculations from before for the heights. I'll try to read the link you sent me (but it doesn't seem as easy as I thought, I'm on vacation until November 6, but I've read the comments :) )

Reply:
> I've kept the calculations from before for the heights. I'll try to read the link you sent me (but it doesn't seem as easy as I thought, I'm on vacation until November 6, butI read the comments :) and your indication to finish this task 😊)

@DanneelsSophie, yeah, implementing this is quite intense. We discussed this at https://github.com/anuraghazra/github-readme-stats/issues/3159#issuecomment-1789079696. I think we will close the `card_height` pull requests for now and migrate to using a `description_lines_count` parameter instead. Sorry for the inconvenience. I will keep this open till the Hacktoberfest has ended 👍🏻.

Reply:
@qwerty541 feel free to close this when you have a `description_lines_count` pull request 👍🏻.